tabs/src/services/lnbitsServiceLocal.ts

```diff
@@ -570,6 +570,29 @@ const getWalletZapsSince = async (
   }
 };
 
+const checkWalletExists = async (
```

Comment:
This should be removed, I don't think it's used, and doesn't call LNBits so wouldn't belong in here anyway.